main.tf

```diff
@@ -100,6 +100,8 @@ resource "aws_cloudfront_distribution" "default" {
 
   cache_behavior = "${var.cache_behavior}"
 
+  web_acl_id = "${var.web_acl_id}"
```

Comment:
will it work if `var.web_acl_id` is empty?
(for compatibility).

Reply:
I was also curious about what will happen if var.web_acl_id is empty. So I did test it.
when var.web_acl_id is empty terraform plan does not even show web_acl_id property being updated. Also tested created cloudfront distribution without setting web_acl_id and with setting web_acl_id. In both cases cloud front distribution was created.